care/emr/reports/context_builder/filters/__init__.py

```
@@ -0,0 +1 @@
+from .tag_filter import *
```

Ruff (0.15.15)

[error] `from .tag_filter import *` used; unable to detect undefined names

(F403)